draft-ietf-suit-firmware-encryption.md

```diff
@@ -508,34 +508,11 @@ The sender needs to execute the following steps:
 
 ### CDDL
 
-The CDDL for the COSE_Encrypt_Tagged structure is shown in {{cddl-aeskw}}.
+The CDDL for the AES-KW binary is shown in {{cddl-aeskw}}.

```

Comment:
The CDDL below is only valid for binaries using AES-KW for key exchange algorithm.